### PR TITLE
Issue50

### PR DESCRIPTION
**Pull request by ahmedqarmout2**
Applying the new design to the website

### CODE REVIEW

File: public/css/admin.css [hunk 1]
```diff
@@ -1,83 +1,4 @@
...
-
-th a:hover {
-  text-decoration: none;
-}
-
-th a:active {
-  color: inherit;
-  text-decoration: none;
-}
+#asd {
```
Comment (ahmedqarmout2) [inline public/css/admin.css]: this is nonsense, i will remove it

File: views/account-edit.pug [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
+          .form-group.row
+            label.col-sm-2.col-form-label(for='points') Points
+            .col-sm-10
+              input.form-control#email(type='number' name='points' value=user.points)
 
-  .form-group.row
-    label.col-sm-2.col-form-label(for='email') Email
-    .col-sm-10
-      input.form-control#email(type='email' name='email' placeholder='New email')
+          .form-group.row
```
Comment (Mohamed-A-Mohamed) [inline views/account-edit.pug]: Indentation is off here, 4 spaces instead of 2. I thought Pug was really picky about this?

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work, looking forward to the knew look.

**Review by Mohammed-JS-Khan — APPROVED** [review]
Looks good to me, Indentation is something we need to look at soon^tm

File: public/js/admin.js [hunk 3]
```diff
@@ -174,8 +171,8 @@ var mcAnswerCount = 4;
 var addMCAnswers = function(dom){
     mcAnswerCount++;
      var newdiv = document.createElement('div');
```
Comment (ahmedqarmout2) [inline public/js/admin.js]: Indentation

File: views/mc-answer.pug [hunk 4]
```diff
@@ -1,11 +1,11 @@
 .form-group
-
 	each index in [1,2,3,4]
 		p
-			| Is Correct:
-			input(type='radio', name='radbutton', value='mcans'+index, required)
+			input(type='radio', name='radbutton', value='mcans'+index, id='mcans'+index required)
```
Comment (ahmedqarmout2) [inline views/mc-answer.pug:4]: Indentation

File: views/question-edit.pug [hunk 5]
```diff
@@ -1,75 +1,72 @@
...
+                #addMoreAnswersButton
+                  button(onclick="addMCAnswers('qAnswer')") add answer
 
-  .form-group.row
-    label.col-sm-2.col-form-label(for='qtitle') Title
-    .col-sm-10
-      input.form-control#qtitle(type='text' name='title' placeholder='New title' value=question.title
-                                autofocus)
+          .form-group.row
+            label.col-sm-2.col-form-label(for='qtopic') Topic
```
Comment (ahmedqarmout2) [inline views/question-edit.pug]: Indentation

File: views/question.pug [hunk 6]
```diff
@@ -17,58 +19,68 @@ block content
...
+            h4.qtext #{question.title}
+          .card-action
+            h5.qtext !{question.text}
+            case question.type
+              when 0
+                form.form-question#re_answerform
+                  label.sr-only(for='answer') Answer
+                  if answered
+                    input.form-control#answer(name='answer' placeholder='Enter your answer here'
+                                              required disabled)
```
Comment (ahmedqarmout2) [inline views/question.pug]: Indentation

File: public/js/admin.js (hunk 3 above)
Comment (Mohammed-JS-Khan) [reply public/js/admin.js]: @ahmedqarmout did you just tell yourself to fix your own PR? xD

**Review by esteeroramy — APPROVED** [review]
Looking good besides the already made indentation comments :)

File: public/js/admin.js (hunk 3 above)
Comment (ahmedqarmout2) [reply public/js/admin.js]: fixed it

File: views/account-edit.pug (hunk 2 above)
Comment (ahmedqarmout2) [reply views/account-edit.pug]: fixed it

File: views/mc-answer.pug (hunk 4 above)
Comment (ahmedqarmout2) [reply views/mc-answer.pug:4]: fixed it

File: views/question-edit.pug (hunk 5 above)
Comment (ahmedqarmout2) [reply views/question-edit.pug]: fixed it

File: views/question.pug (hunk 6 above)
Comment (ahmedqarmout2) [reply views/question.pug]: fixed it

**Pull request merged**